src/Provider/Youtube/Provider.php

```diff
@@ -135,35 +135,26 @@ public function provide($input)
         // thanks to amit kumar @ bloggertale.com for sharing the fix
 		
 		//This URL *should* allow age restricted videos and regular videos to download
-        $video_info_url = 'https://www.youtube.com/get_video_info?&video_id=' . $input . '&asv=3&hl=en_US';
-
-        $request = $this->getHttpClient()->createFullRequest(
-            'GET',
-            $video_info_url
-        );
+        //$video_info_url = 'https://www.youtube.com/get_video_info?&video_id=' . $input . '&asv=3&hl=en_US';
 
         $options = ['curl' => []];
 
         if ($this->options['use_ip'] !== false) {
             $options['curl'][CURLOPT_INTERFACE] = $this->options['use_ip'];
         }
 
+        $video_info_url = 'https://www.youtube.com/get_video_info?&video_id=' . $input . '&asv=3&el=detailpage&hl=en_US';
+        $request = $this->getHttpClient()->createFullRequest(
+            'GET',
+            $video_info_url
+        );
         $response = $this->getHttpClient()->send($request, $options);
 		
-		//But just incase it doesn't, we can fallback to the old URL.
-		if (strpos($response->getBody()->__toString(), 'status=fail') !== false) {
-			$video_info_url = 'https://www.youtube.com/get_video_info?&video_id=' . $input . '&asv=3&el=detailpage&hl=en_US';
-            $request = $this->getHttpClient()->createFullRequest(
-                'GET',
-                $video_info_url
-            );
-            $response = $this->getHttpClient()->send($request, $options);
```

Comment:
This fallback was introduced with #365. Why do you deleted it?

Reply:
This fallback is falling back on the URL that is the only one actually working right now, therefore I commented out the old URL and substituted it with the new one and deleted the fallback. The benefit of getting age restricted videos is no longer there if no videos are working at the moment.